### PR TITLE
Withdraw endpoint

### PR DESCRIPTION
**Pull request by ramontayag**

### CODE REVIEW

**Comment by mrkjlchvz** [comment]
Btw, the table `withdrawal_requests` uses `asset_type` instead of `type`. According to SEP-006's standard, we should use `type` for this. Is it like this so that we avoid unexpected inheritance issues since this will automatically allow an STI setup?

**Comment by mrkjlchvz** [comment]
Naming suggestion: `c.withdraw` to `c.withdrawable_assets`?

File: stellar_base.gemspec [hunk 1]
```diff
@@ -31,5 +31,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'vcr', '~> 4.0'
   s.add_development_dependency 'webmock'
   s.add_development_dependency 'factory_bot_rails'
+  s.add_development_dependency "stellar-sdk"
```
Comment (mrkjlchvz) [inline stellar_base.gemspec]: Any reason why this is added only in development?

Comment (ramontayag) [reply stellar_base.gemspec]: That's because at this point, we don't need to do any calls to the stellar network from Ruby

**Comment by ramontayag** [comment]
https://github.com/bloom-solutions/stellar_base-rails/pull/10#issuecomment-414051935

That's right. But `type` is a reserved ActiveRecord word. We respond with [type](https://github.com/bloom-solutions/stellar_base-rails/pull/10/files#diff-99189ceb68994efb9203958ec76fbf13R37), though. We just save it as `asset_type` in the db.

**Comment by ramontayag** [comment]
https://github.com/bloom-solutions/stellar_base-rails/pull/10#issuecomment-414052690

I like that. Will edit

**Pull request merged**